e2e/app/admin/common.go

```diff
@@ -131,15 +135,17 @@ func report(ctx context.Context, results runResults) error {
 		}
 	}
 
-	if len(failed) == len(results) {
+	if len(failed) == runs {
```

Comment:
Not clear what we're gaining by introducing `runs`, seems like `len(results)` was achieving the same thing?

Reply:
`len(results)` was always zero. I assume cause it's something like an iterator that's been spent? though haven't confirmed

Reply:
(discussed offline)

ah, it's a channel, that makes sense